add-ons/hyper-sorter/index.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+var _ = require('object-iterators');
+
+function Hypersorter(grid, objects) {
+    this.grid = grid;
```

Comment:
make this a simple function. doesn't need to be an object with no methods or props:
```javascript
module.exports = function(grid, objects) {
```